### DISCUSSION TITLE
2007/10/31/%E7%A7%9F%E6%88%BF%E6%97%B6%E4%BB%A3%E7%9A%84%E5%9B%9E%E9%A1%BE

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#187] [Announcements]
# 2007/10/31/%E7%A7%9F%E6%88%BF%E6%97%B6%E4%BB%A3%E7%9A%84%E5%9B%9E%E9%A1%BE

周末的时候与房东结清了水电费、押金等所有的帐目，于是正式结束了我一段长达八年的租房生活。为了庆祝我生命中下一个崭新篇章-房奴时代的开始，我决定好好回想一下这些年来与租房相关的经历。

https://blog.qizhen.xyz/2007/10/31/%E7%A7%9F%E6%88%BF%E6%97%B6%E4%BB%A3%E7%9A%84%E5%9B%9E%E9%A1%BE

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Hongbo
2008年09月27日 11:13 编辑
恭喜！好得翻身“农奴”作主人了！

回复
Sola
2008年08月18日 16:51 编辑
我是该恭喜呢还是该同情呢？！！！
恭喜你有了自己的房子
同情这个社会

回复
Justin
2008年01月2日 09:58 编辑
还行

回复
Xiaohong
2007年12月4日 16:48 编辑
我也觉得这是该blog所有文章中水平最高的一篇了.  不过看了健键的评论我才知道你连打游戏都这么自私啊，都不救同伴的，biangji~

回复
6
2007年11月7日 17:55 编辑
其实我也想知道你是谁…天天在我的好友列表看到你的SPACE..

回复
Gladys Pang
2007年11月5日 12:45 编辑
写的真好玩。要求参观新居。。

回复
幽灵公主
2007年11月4日 19:22 编辑
haha  不也不知道你是谁 不过在我的blog留言了 我便吸引过了
这个文章太好玩了
你的洗衣机多少钱买的

回复
千纤草
2007年11月1日 15:13 编辑
哈哈，翻身“农奴”做主人了~^^

回复
yoyo
2007年10月31日 20:39 编辑
八年抗战 终于胜利拉 恭喜~

回复
arrowpig
2007年10月31日 13:11 编辑
不过，比起打星际，小黑店就不算什么了，阮奇桢只会用人族，lost temper，一上来就在家门口抱两个地保，然后只顾自己发展，造坦克，从来不去救同伴。要就也是派两个小宾意思意思，诶，别人灭了他的同伴就去灭他。所以阮奇桢经常不是被打死的，而是没有钱，穷死的。他只组里唯一一个我可以打赢得人，只有那么一次，我跟他担挑，我占领了其他所有的矿产，他还老强调，门口地堡。我想算了，反正我钱多，就送了很多dragon送死，结果他家里还有很多航母，我来不及造兵，被他打败了。就这唯一的一次，但是以后阮奇桢就说他打败了我。所以后来我打败了李楠，并且死不承认自己星际水平比李楠差，就是跟阮奇桢学的~~~

回复
arrowpig
2007年10月31日 12:47 编辑
恩，同意永青，那次还是我们一起去帮忙搬家的呢。想我当年到NI的第一顿中饭去了味之都。第二顿饭到了小黑店，阿，小黑店的饭可真好吃阿，比交大的饭好吃很多，于是我从早上上班的时候就开始准备吃中午饭了。小黑店的电饭煲太小了，于是我掌握的承饭的权利，每个人吃多少都是由我决定的哦，那个时候阮奇桢就是喜欢吃虾，这家伙拨虾的速度叹为观止阿！可惜后来他皮肤过敏，他自己调侃自己的了不治之症。诶，吴店长还是很慷慨的，我们吃的很好，后来王店长就抠门多了。跟老外介绍small black** 的时候，诶，我那个中国英语阿，阮奇桢后来跟我说这是有典故的，据说宋朝的时候…..

回复
永青
2007年10月31日 10:30 编辑
经鉴定：
1）这是我看到阮大侠写出的最为幽默的一篇文章
2）没想到还会弹吉他，一直以为只会弹棉花
3）当时刚进公司，看到阮大侠住在那650元月租的车棚里，我的心彻底的凉了。。。
 

回复
fang
2007年10月31日 09:58 编辑
你的租房史都快赶上8年抗战了。。。
不过，终于有了自己的房子，恭喜了，呵呵。

回复
妍
2007年10月31日 09:53 编辑
一直在想，你是哪位，在我的好友列表里面，但我想不出来你是谁…..

回复
arrowpig
2007年10月31日 09:01 编辑
好心切呀，现在上班时间，晚上下班了再来回忆我们小黑店的日子哦~~

回复
